text/70_performance_testing.md

```diff
@@ -14,13 +14,13 @@ As mentioned above, defining performance for a particular system means determini
 
 A __service-oriented indicator__ measures how well a system is providing a service to a particular user.  These are measured from the user's perspective and quantify aspects of the system the user would directly care about.  Some examples of service-oriented indicators would be average page load time, response time, or what percentage of the time the system is available.
 
-There are two main subcategories of service-oriented indicators: availability, or how available is the system to the user.  This could mean anything from what percentage of the time can the system be accessed to how available different features are to the users at different times.  The second main category is response time - how long does it take for the system to respond to the user's input or return a result?
+There are two main subcategories of service-oriented indicators. The first is availability - how available is the system to the user?  This could mean anything from what percentage of the time can the system be accessed to how available different features are to the users at different times.  The second main category is response time - how long does it take for the system to respond to the user's input or return a result?
```

Comment:
The way the first sentence leads off ("There are two main subcategories of service-oriented indicators:") made me think that "availability" was the first indicator and "how available is the system to the user" was the second. I had to stop for a second to figure out what was going on. I suggest this more parallel construction to make the structure more clear.